src/components/ui/SearchForm.vue

```diff
@@ -141,20 +149,7 @@ export default {
 .search-form__input::placeholder {
   text-align: center;
 }
-.search-form__input:focus,
-.search-form__input:hover {
-  border: 1px solid var(--blue-60);
-}
-.search-form__input:focus + .search-form__submit,
-.search-form__input:hover + .search-form__submit {
-  border-color: var(--blue-60);
-}
-.focus-styles .search-form__input:focus,
-.focus-styles .search-form__input:hover,
-.focus-styles .search-form__input:focus + .search-form__submit,
-.focus-styles .search-form__input:hover + .search-form__submit {
-  border-color: var(--gray-30);
```

Comment:
I think these classes are used to remove the blue border if in _tab_ mode (See App.vue) to not confuse the user what is actually selected.

Reply:
After confirming this behaviour in the UI, it appears that this is not an issue. I've attached the behavior to the jira story.

Reply:
Also, removing this seems to be necessary as it seems to keep the border gray even on hover after tabbing.

Reply:
Cool